front/app/components/InitiativeMap/components/InitiativeInformationOverlay.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+import eventEmitter from 'utils/eventEmitter';
+import InitiativePreview from './InitiativePreview';
+import CloseIconButton from 'components/UI/CloseIconButton';
+import styled from 'styled-components';
+
+type Props = {
+  modalPortalElement: HTMLDivElement;
+};
+
+const StyledCloseButton = styled(CloseIconButton)`
+  width: 100%;
+  justify-content: flex-end;
+  padding-top: 8px;
+  padding-right: 8px;
+`;
+
+const InitiativeInformationOverlay = ({ modalPortalElement }: Props) => {
+  const [selectedInitiativeId, setSelectedInitiativeId] = useState<
+    string | null
+  >(null);
+
+  // Listen for initiative selection event and set selected ID
+  useEffect(() => {
+    const subscription = eventEmitter
+      .observeEvent('initiativeSelectedEvent')
+      .subscribe(({ eventValue }) => {
+        setSelectedInitiativeId(eventValue as string);
+      });
+
+    return () => {
+      subscription.unsubscribe();
+    };
+  }, []);
+
+  if (!modalPortalElement) return null;
+  if (!selectedInitiativeId) return null;
+
+  return createPortal(
```

Comment:
I think usually we use`createPortal` for full screen stuff like the content builder or modals... why are we using it here? From what I understand, this overlay needs to be on top of the map, right? I think in that case it can also be solved by using `zIndex`, `position: absolute` and `left / right / top / bottom` properties. That way this will be rendered just on top of the map, but it will still be possible to overlay other things over the map- now it will be rendered on top of our entire application